src/lab14.py

```diff
@@ -445,16 +445,7 @@ def layout(self, zoom):
         child.layout(zoom)
         self.height = child.height + 2* device_px(VSTEP, zoom)
 
-    def paint(self, display_list, dark_mode):
-        if dark_mode:
-            background_color = "black"
-        else:
-            background_color = "white"
-        display_list.append(
-            DrawRect(skia.Rect.MakeLTRB(
-                self.x, self.y, self.x + self.width,
-                self.y + self.height),
-                background_color, background_color))
+    def paint(self, display_list):
```

Comment:
You didn't indicate this change in your chapter text edits. How about just editing earlier chapters to remove the white background from DocumentLayout's paint method and leave it in draw?

Reply:
This is a great find because it turns out we added it in Chapter 13 without explaining it.